faststream/nats/subscriber/usecase.py

```diff
@@ -418,13 +426,54 @@ def __init__(
             include_in_schema=include_in_schema,
         )
 
+    async def get_one(
+        self,
+        *,
+        timeout: float = 5.0,
+    ) -> "Optional[NatsMessage]":
+        assert self._connection, "Please, start() subscriber first"
+        assert (  # nosec B101
+            not self.calls
+        ), "You can't use `get_one` method if subscriber has registered handlers."
+
+        if self.subscription is None:
+            self.subscription = await self._connection.subscribe(
+                subject=self.clear_subject,
+                queue=self.queue,
+                **self.extra_options,
+            )
+
+        raw_message = None
+
+        with anyio.move_on_after(timeout):
+            async for raw_message in self.subscription.messages:
```

Comment:
Why not `self.subscription.get_one`?

Reply:
Fixed

Reply:
What's the difference between `get_one` method and `next_msg`? 😃